tests/test_distributions.py

```diff
@@ -113,6 +122,16 @@ def test_contains():
     assert iu._contains(10.1)
     assert not iu._contains(11)
 
+    # IntUniformDistribution with a 'q' parameter.
+    iuq = distributions.IntUniformDistribution(low=1, high=10, q=2)
+    assert not iuq._contains(0.9)
+    assert iuq._contains(1)
+    assert iuq._contains(3.5)
```

Comment:
why does an int distribution contain a non-int?

Reply:
@cthoyt 
According to @c-bata's review, 

> It's ok to just check the sampled value is in [low, high] for now. At least, this behavior is the same with DiscreteUniformDistribution._contains. 

https://github.com/optuna/optuna/pull/910#pullrequestreview-355781758

Reply:
On second thought, I think it should not be checked in the test case. Because it's just implementation behavior, not defined as the specification. 

@hayata-yamamoto How about updating a test case like the following? I'm sorry that the code is originally added by me.

```python
    iuq = distributions.IntUniformDistribution(low=1, high=10, q=2)
    assert not iuq._contains(0)
    assert iuq._contains(1)
    assert iuq._contains(4)  # (※1)
```

(※1) Although `IntUniformDistribution(low=1, high=10, q=2)` might be `[1, 3, 5, 7, 9]`, `4` should be True. Because the sampler might sample from `[2, 4, 6, 8, 10]`.

Reply:
@c-bata 
> How about updating a test case like the following? I'm sorry that the code is originally added by me.

Okay! I'll do that.


Anyway, 

> Although IntUniformDistribution(low=1, high=10, q=2) might be [1, 3, 5, 7, 9], 4 should be True. Because the sampler might sample from [2, 4, 6, 8, 10].

Could you explain the above in a bit more detail? 
It was understood that `_contains` method judges **whether the value is included in [low, high] or not. It's not considered whether the value is in sampling space or not**. 

Is this interpretation correct?

Reply:
> It was understood that _contains method judges whether the value is included in [low, high] or not. It's not considered whether the value is in sampling space or not.

Yes. That's what I mean.